extensions-core/kinesis-indexing-service/src/main/java/org/apache/druid/indexing/kinesis/supervisor/KinesisSupervisor.java

```diff
@@ -212,14 +224,27 @@ protected void scheduleReporting(ScheduledExecutorService reportingExec)
     // not yet implemented, see issue #6739
   }
 
+  /**
+   * We try to parse the shard number of the shard ID, using a BigInteger because the Kinesis shard ID can be
```

Comment:
looks like this comment is no longer true

Reply:
whoops, fixed, thanks